jerry-core/parser/js/js-parser-expr.c

```diff
@@ -64,17 +64,21 @@ static const uint8_t parser_binary_precedence_table[] =
 #if ENABLED (JERRY_ESNEXT)
   3,
 #endif /* ENABLED (JERRY_ESNEXT) */
-  4, 5, 6, 7, 8, 9, 10, 10, 10, 10,
-  11, 11, 11, 11, 11, 11, 12, 12, 12,
-  13, 13, 14, 14, 14,
+  4,
 #if ENABLED (JERRY_ESNEXT)
-  15,
+  5,
+#endif /* ENABLED (JERRY_ESNEXT) */
+  6, 7, 8, 9, 10, 11, 11, 11, 11,
+  12, 12, 12, 12, 12, 12, 13, 13, 13,
+  14, 14, 15, 15, 15,
+#if ENABLED (JERRY_ESNEXT)
```

Comment:
Please separate these into single lines with a comment about the corresponding binary operator.

Reply:
Great idea!